CHANGELOG.md

```diff
@@ -4,8 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.7.1] - 2022-04-10
+## [0.8.0] - 2022-04-19
```

Comment:
could you replace this line with `## [Unreleased]` (just to follow what's been done till now).
Btw, there's no breaking change right, we'll be able to release a minor?

Reply:
No there is no breaking change but looking at the changelog it seems that every addition increase the minor